ml_enabler/api/ml.py

```diff
@@ -1,7 +1,67 @@
-from flask_restful import Resource
-
+from flask_restful import Resource, request, current_app
+from ml_enabler.models.dtos.ml_model_dto import MLModelDTO
+from schematics.exceptions import DataError
+from ml_enabler.services.ml_model_service import MLModelService
+from ml_enabler.models.utils import NotFound
+from sqlalchemy.exc import IntegrityError
 
 class StatusCheckAPI(Resource):
+    """ Healthcheck method """
 
     def get(self):
         return {'hello': 'world'}, 200
+
+
+class MLModelAPI(Resource):
+
+    def post(self):
+        """ Subscribe a new ML model """
+        try:
+            model_dto = MLModelDTO(request.get_json())
+            current_app.logger.info(f'request: {str(request.get_json())}')
+            model_dto.validate()
+            model_id = MLModelService.subscribe_ml_model(model_dto)
+            return {"model_id": model_id}, 200
+        except DataError as e:
+            current_app.logger.error(f'Error validating request: {str(e)}')
+            return str(e), 400
+        except IntegrityError as e:
+            current_app.logger.error(f'A model with the same name already exists: {str(e)}')
+            return str(e), 400
+
+    def delete(self, model_id):
```

Comment:
Can you add docstrings for this and `get` and include a few words about the parameter values

Reply:
Yes I'll add docstrings to all the API methods.